src/main/java/ch/wisv/areafiftylan/seats/model/Seat.java

```diff
@@ -39,7 +39,7 @@
     private Long Id;
 
     @JsonView(View.Public.class)
-    private boolean taken = false;
```

Comment:
Please check whether we use the taken property in the frontend, changes may be needed there.